app/ide-desktop/lib/dashboard/package.json

```diff
@@ -3,13 +3,21 @@
   "version": "0.1.0",
   "type": "module",
   "private": true,
+  "comments": {
+    "playwright": "If using a tiling window manager, ensure that the Playwright windows are",
+    "playwright (cont.)": "not tiled, otherwise the screenshots will be incorrect.",
+    "FIXME[sb]": "Add `npm run test-e2e` back to the 'test' script when actions are fixed?"
```

Comment:
`FIXME[sb]` -> `FIXME [sb]`

Reply:
Looks like it's part of the `test` script again so this comment can be removed?

Reply:
yeah. the other comments can too, since i switched from headful (because headless wasn't properly rendering `backdrop-filter: blur`) back to headless